lib/utils/errors.js

```diff
@@ -23,6 +23,34 @@ class DecryptError extends EvervaultError {}
 
 class RelayOutboundConfigError extends EvervaultError {}
 
+class UnauthorizedError extends EvervaultError {}
```

Comment:
I have that already 🙂 Because these errors extend `EvervaultError` we inherit the constructor with the `message` param which I use any time we initialise an error 👍